src/web/nextui/src/app/eval/Eval.tsx

```diff
@@ -124,13 +124,14 @@ export default function Eval({
     } else if (fetchId) {
       console.log('Eval init: Fetching eval', fetchId);
       const run = async () => {
-        const host = IS_RUNNING_LOCALLY ? REMOTE_API_BASE_URL : '';
-        const url = `${host}/api/eval/${fetchId}`;
-        console.log('Fetching eval from remote server', url);
-        const response = await fetch(url);
+        console.log('Fetching eval from remote server', fetchId);
+        let response = await await callApi(`/eval/${fetchId}`);
         if (!response.ok) {
-          setFailed(true);
-          return;
+          response = await callApi(`/results/${fetchId}`);
```

Comment:
Try both endpoints for now, until we clean this up

Reply:
This allows you now access any eval like this:

`http://localhost:3100/eval/eval-2024-09-06T23:05:53`